_posts/2017-10-20-the-data-and-science-behind-grabshare-part-i.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+id: the-data-and-science-behind-grabshare-part-i
+title: "The Data and Science Behind GrabShare Part I: Verifying potential and developing the algorithm"
+date: 2017-10-20 07:30:40
+authors: [muchen-tang]
+categories: [Data Science]
+tags: [Data Science, GrabShare]
+comments: true
+cover_photo: /img/the-data-and-science-behind-grabshare-part-i/cover.png
+excerpt: "Launching GrabShare was no easy feat. After reviewing the academic literature, we decided to take a different approach and build a new matching algorithm from the ground up."
+---
+
+![cover](/img/the-data-and-science-behind-grabshare-part-i/cover.png)
```

Comment:
I think don't have to show the cover photo here because there is the original one below

Reply:
uhm... but it would look nice to have a cover photo at top of the post as part of layout?

Reply:
never mind, i think removing is also fine.

Reply:
I think it's ok. In Medium, the cover photo for the post is basically selecting one of the images used on the page.